File: testdata/bench/lua/BenchmarkTest00009.lua
local pgmoon = require "pgmoon"
local args = ngx.req.get_post_args()
local username = args.username
local email = args.email
local pg = pgmoon.new({host="127.0.0.1", database="app"})
pg:connect()
local q = string.format("INSERT INTO users (name, email) VALUES ('%s', '%s')", username, email)
pg:query(q)
ngx.say("created")

File: testdata/bench/csharp/BenchmarkTest00009.cs
using System.Data.SqlClient;
using Microsoft.AspNetCore.Mvc;
using System;

namespace BenchApp.Controllers
{
    public class SqliController : Controller
    {
        [HttpPost]
        public IActionResult Insert()
        {
            string name = Request.Form["name"];
            string email = Request.Form["email"];
            var conn = new SqlConnection("Server=localhost;Database=app;");
            conn.Open();
            var sql = String.Format("INSERT INTO users (name, email) VALUES ('{0}', '{1}')", name, email);
            var cmd = new SqlCommand(sql, conn);
            cmd.ExecuteNonQuery();
            return Ok();
        }
    }
}
